core-library/src/main/java/org/silverpeas/core/i18n/BeanTranslation.java

```diff
@@ -100,9 +100,9 @@ public final void setDescription(String description) {
   }
 
   @Override
-  protected Translation clone() {
+  protected BeanTranslation clone() {
     try {
-      return (Translation) super.clone();
+      return (BeanTranslation) super.clone();
     } catch (CloneNotSupportedException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1166)